paddle/phi/api/yaml/ops.yaml

```diff
@@ -364,6 +384,26 @@
     backend : x
   inplace: (x -> out)
 
+- op : bitwise_right_shift_arithmetic
+  args : (Tensor x, Tensor y)
+  output : Tensor(out)
+  infer_meta :
+    func : ElementwiseInferMeta
+  kernel :
+    func : bitwise_right_shift_arithmetic
+    backend : x
+  inplace: (x -> out)
+
+- op : bitwise_right_shift_logic
+  args : (Tensor x, Tensor y)
+  output : Tensor(out)
+  infer_meta :
+    func : ElementwiseInferMeta
+  kernel :
+    func : bitwise_right_shift_logic
+    backend : x
+  inplace: (x -> out)
+
```

Comment:
According to the [C++ operator development specifications](https://www.paddlepaddle.org.cn/documentation/docs/zh/dev_guides/api_contributing_guides/new_cpp_op_cn.html#qi-kaifasuanzizhuyishixiang), the naming and parameters of operator need to be consistent with Python API, so that operator can reuse documentation of Python API. When user writes  C++ code directly using operator, it will be relatively simple. So the operator needs to be `bitwise_right_shift` with `is_arithmetic` in args, kernels in file [paddle/phi/kernels/bitwise_kernel.h](https://github.com/PaddlePaddle/Paddle/pull/58092/files#diff-35dca83d346965fa32eb7a2bd1cd0fa6a09a9853fba18d9a38269257cb421894) should also have`is_arithmetic` in args. and it's best if Functor in [paddle/phi/kernels/funcs/bitwise_functors.h](https://github.com/PaddlePaddle/Paddle/pull/58092/files#diff-9852da3163fbef8a980df35d58e40d115076ed3eb511b020a76963c03d68a2c6) can be unified, but it's okay if it can't be unified.